commcare-cloud/commcare_cloud.py

```diff
@@ -247,24 +252,65 @@ def make_parser(parser):
         parser.add_argument('-u', '--user', default='ansible', help=(
             "The user to run the commands as."
         ))
+        parser.add_argument('--become', action='store_true', help=(
```

Comment:
could you add the short form as well: `'-b'`

Reply:
oops, I should have waited to merge until @dannyroberts had a chance to respond to this (probably by adding 4 characters to this PR).

Reply:
Ah, yes, of course. Thanks, will do!

Reply:
https://github.com/dimagi/commcarehq-ansible/pull/1112